platoon_strategic_IHP/config/parameters.yaml

```diff
@@ -0,0 +1,19 @@
+maxPlatoonSize: 10
```

Comment:
Each variable here should have a comment that describes its purpose and its units of measure.  If there are upper and lower limits on values it can be assigned, those should also be noted.